how-to/how-to-rename-field-in-content-type.md

```diff
@@ -0,0 +1,50 @@
+---
+title: How to Rename a Field in Content Type
+description: Rename a field in content type
+type: how-to
+page_title: How to Change the Name of a Field in a Content Type
+slug: rename-field-in-content-type
+position:
+tags: content-type
+teampulseid: 
+ticketid:  1046311
+pitsid:
+
+---
+                    
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>{{site.bs-d}} ({{site.bs}})</td>   
+  </tr>
+</table>
+
+## Description
+You want to rename an existing field in a content type. The rename option is not available in {{site.tp}} UI. 
+
+## Solution
+Each item in a content type is stored as a key-value pair ({Field-Name: Field-Value}). Therefore when you want to rename the field name you need to change all keys in the database ("Field-Name" with "New-Field-Name").
+
+>As this is a database operation, please note that updating the keys in the data will update also:
+> * "ModifiedAt" value will be updated with current date for all renamed items
+> * "ModifiedBy" value will be nullified for all renamed items
+
+To rename a field you have to complete the following steps:
+1. Step 1 - Update the keys in the NoSQL database - you can use '$rename' operator and issue a PUT request:

```

Comment:
Step 1 and Step 2 - omit them. NoSQl - too. Update the field name in the database.